test/connector.test.js

```diff
@@ -103,5 +103,11 @@ describe('Connector', () => {
           'non.existing.property'
         )).to.throw(/Invalid property path/);
     });
+
+    it('should preserve backward-compatibility for non-existing property', () => {
+      const definition = connector.getPropertyDefinition('MyModel', 'idontexist');
+      // eslint-disable-next-line no-unused-expressions
+      expect(definition).to.be.undefined;
```

Comment:
Does it allow `expect(definition).to.be.undefined();`?

Reply:
I tried, but it doesn't according to the [docs](https://www.chaijs.com/api/bdd/#method_undefined)

Reply:
@b-admike Have you considered the following?

```js
expect(definition).to.equal(undefined);
```

[Beware of libraries that assert on property access!](https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access)

In the past, I have had good experience with using [dirty-chai](https://www.npmjs.com/package/dirty-chai) plugin for `chai`, see how we are using it in LoopBack 3.x here: [loopback's test/helpers/expect.js](https://github.com/strongloop/loopback/blob/c8a31e2725a97ef415604492f38d60178131478b/test/helpers/expect.js)